src/main/cpp/libclp_ffi_java/Java_EightByteClpWildcardQueryEncoder.cpp

```diff
@@ -127,7 +128,7 @@ static jobjectArray encode_native (
         for (const auto& query_var : query_vars) {
             auto success = std::visit(overloaded{
```

Comment:
```suggestion
            const auto success = std::visit(overloaded{
```

Reply:
Let's defer this to another PR where I apply our clang-format and clang-tidy settings if possible.